Site/css/style.css

```diff
@@ -586,14 +591,19 @@ form textarea {
 .card-name {
 	color: #4d3142;
 	font-weight: bold;
-	font-size: 125%;
+	/* font-size: 125%; */
```

Comment:
Bare for å se hva som skjer? Burde enten være med eller fjernes helt.

Reply:
da er det en del andre kommenterte ut ting her og der som burde sjekkes. så kjapt på det nå, det gjorde ingenting da, hadde kanskje noe (mer) effekt før bold kjørte ordentlig